lib/cache/cache_test.go

```diff
@@ -742,6 +742,63 @@ func benchGetNodes(b *testing.B, nodeCount int) {
 	}
 }
 
+/*
+goos: linux
+goarch: amd64
+pkg: github.com/gravitational/teleport/lib/cache
+cpu: Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
+BenchmarkListResourcesWithSort-8               1        2351035036 ns/op
+*/
+func BenchmarkListResourcesWithSort(b *testing.B) {
+	p, err := newPack(b.TempDir(), ForAuth, memoryBackend(true))
+	require.NoError(b, err)
+	defer p.Close()
+
+	ctx := context.Background()
+
+	count := 100000
+	for i := 0; i < count; i++ {
+		func() {
```

Comment:
Why do you need the anonymous function here as opposed to just running this code in the loop directly?

Reply:
tbh i didn't pay attention to it, i asked the original author and they don't remember, so i removed it